content/faq.md

```diff
@@ -2,25 +2,27 @@
 title = "Frequently Asked Questions (FAQ)"
 +++
 
+**This article has been verified for VCMI 1.1 (release date 23.12.2022)**
+
 * Again, what is VCMI?
 
-> In simple words: Remake of Heroes of Might and Magic 3. We use original Heroes 3 graphic, sound and text assets and make new game that is supposed to work 100% same as H3. Everything in VCMI is made from scratch - loading game files, drawing graphics, AI, game mechanics etc. We also plan supporting WoG mod features, but for now there are areas in VCMI that need more attention than WoG features.
+> In simple words: Remake of Heroes of Might and Magic 3. We use original Heroes 3 graphic, sound and text assets and make new game that is supposed to work 100% same as H3. Everything in VCMI is made from scratch - loading game files, drawing graphics, AI, game mechanics etc. We also plan supporting WoG, HD mod and other cool features, but there are other areas that need our attention as well.
 
 * What is the purpose of VCMI? Why to play or develop it if Heroes 3 is already there?
 
-> Our goal is to make "better" heroes 3 that overcomes various H3 engine limitations and allows way easier extensibility / modding than original game. Also VCMI works without problems on Windows, Linux, MacOS and Android.
+> Our goal is to make "better" heroes 3 that overcomes various H3 engine limitations and allows way easier extensibility / modding than original game. Also VCMI is multiplatform so it works on Windows, Linux, MacOS, iOS and Android.
```

Comment:
VCMI already allows that for a long time, and there are plenty of mods already up and running.

Reply:
It allows modding, but not to some incredibly good extent + it isn't exactly "ultimate better H3 yet" that meets goals such as superceding original WoG/ERA or beingundoubtful better base for H3 than reverse engineering, considering that VCMI flaws add more uncertainty in this regard.